src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/model/SendApplicationToPrintMessageBuilder.kt

```diff
@@ -0,0 +1,48 @@
+package uk.gov.dluhc.printapi.testsupport.testdata.model
+
+import uk.gov.dluhc.printapi.messaging.models.CertificateDelivery
+import uk.gov.dluhc.printapi.messaging.models.CertificateLanguage
+import uk.gov.dluhc.printapi.messaging.models.ElectoralRegistrationOffice
+import uk.gov.dluhc.printapi.messaging.models.SendApplicationToPrintMessage
+import uk.gov.dluhc.printapi.messaging.models.SourceType
+import uk.gov.dluhc.printapi.testsupport.testdata.DataFaker.Companion.faker
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidApplicationReference
+import uk.gov.dluhc.printapi.testsupport.testdata.getIerDsApplicationId
+import uk.gov.dluhc.printapi.testsupport.testdata.getRandomEro
+import uk.gov.dluhc.printapi.testsupport.testdata.getRandomGssCode
+import java.time.Instant
+import java.time.OffsetDateTime
+import java.time.ZoneOffset.UTC
+import java.util.UUID
+
+fun buildSendApplicationToPrintMessage(
+    sourceReference: String = getIerDsApplicationId(),
+    applicationReference: String = aValidApplicationReference(),
+    sourceType: SourceType = SourceType.VOTER_MINUS_CARD,
+    issuingAuthority: String = getRandomEro(),
+    requestDateTime: OffsetDateTime = Instant.now().atOffset(UTC),
+    firstName: String = faker.name().firstName(),
+    middleNames: String = faker.name().firstName(),
```

Comment:
Thanks, @vishalgupta-valtech.
 
I don't know the reason but `middleNames` is defined as a required field in the openApi specs, and thus generated as String. That's why it is not nullable here.

<pre>
    @get:Size(max=255)
    @Schema(example = "Malcolm", <b>required = true</b>, description = "Middle names of the Elector")
    @field:JsonProperty("middleNames", <b> required = true </b>) val middleNames: <b>kotlin.String</b>,
</pre>

Reply:
Interesting, I guess the value must be included in the CSV file, but presumably it can be left blank.  So VCA will be responsible for mapping `null` to `""` when posting to this queue. 👍